src/main/java/com/gdschongik/gdsc/domain/member/domain/Member.java

```diff
@@ -86,4 +89,15 @@ public static Member createGuestMember(String oauthId) {
                 .status(MemberStatus.NORMAL)
                 .build();
     }
+
+    public void softDelete() {
```

Comment:
```suggestion
    public void withdraw() {
```

의미를 더 잘 드러낼 수 있는 메서드 이름을 써주면 좋을 것 같습니다.

Reply:
좋은 지적 감사합니다👍